### PR TITLE
Add CI job for testing gce-pd-csi driver for windows

### PR DESCRIPTION
**Pull request by jingxu97**
Add ci job config file

### CODE REVIEW

**Comment by jingxu97** [comment]
cc @msau42

File: config/jobs/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/gcp-compute-persistent-disk-csi-driver-windows.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      args:
+      - --check-leaked-resources
+      - --cluster=
+      - --extract=ci/k8s-master
+      - --gcp-zone=us-west1-b
+      - --ginkgo-parallel=8
+      - --provider=gce
+      - --gcp-nodes=2
+      - --test=false
+      - --test-cmd=$GOPATH/src/sigs.k8s.io/gcp-compute-persistent-disk-csi-driver/test/run-windows-script.sh
```
Comment (msau42) [inline config/jobs/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/gcp-compute-persistent-disk-csi-driver-windows.yaml:29]: This should be run-windows-k8s-integration.sh?

Comment (jingxu97) [reply config/jobs/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/gcp-compute-persistent-disk-csi-driver-windows.yaml]: fixed, thanks!

File: config/jobs/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/gcp-compute-persistent-disk-csi-driver-windows.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      - --check-leaked-resources
+      - --cluster=
+      - --extract=ci/k8s-master
+      - --gcp-zone=us-west1-b
+      - --ginkgo-parallel=8
+      - --provider=gce
+      - --gcp-nodes=2
+      - --test=false
+      - --test-cmd=$GOPATH/src/sigs.k8s.io/gcp-compute-persistent-disk-csi-driver/test/run-windows-k8s-integration.sh
+      - --test-cmd-args=--ginkgo.focus=External.Storage.*ntfs --ginkgo.skip=\[LinuxOnly\]|\[Serial\]|\[Feature:.+\] --minStartupPods=8
```
Comment (msau42) [inline config/jobs/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/gcp-compute-persistent-disk-csi-driver-windows.yaml:30]: Are these args correct?

Comment (jingxu97) [reply config/jobs/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/gcp-compute-persistent-disk-csi-driver-windows.yaml]: removed them. the script already has all the args.

**Comment by jingxu97** [comment]
@msau42 I also added a submit job. PTAL.

**Pull request merged**